controller/controller.js

```diff
@@ -11,6 +11,18 @@ const getTopics = (req,res,next) => {
     
 }
 
+const getCommentsByArticleId = (req, res, next) => {
+  const { article_id } = req.params;
 
+  if (isNaN(article_id)) {
```

Comment:
Remove this whole if statement and make sure, you are handling the psql error and sending back the appropriate message.